### PR TITLE
doc(components/Dialog): fix 'show info' button in storybook

### PR DESCRIPTION
**Pull request by rmannibucau**
Thanks Fabien for the fix tip

**What is the problem this PR is trying to solve?**

No way to click on "show info" in the storybook of the dialog component

**What is the chosen solution to this problem?**

pointer-events: none in the css of the dialog for the story book

**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/stories/Dialog.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+div[role="dialog"] {
```
Comment (frassinier) [inline packages/components/stories/Dialog.css]: Good idea, but I'm not an huge fan of adding CSS next to our stories.

Just add this JSX in _packages/components/stories/Dialog.js_
```javascript
storiesOf('Dialog', module)
	.addDecorator(story => (
		<div>
			<style>{"\
				.modal, .modal-backdrop {\
					pointer-events: none;\
				}\
				.modal-dialog {\
				    pointer-events: auto;\
			    }\
			"}</style>
			{story()}
		</div>
	))
```

Comment (talend-glorieux) [reply packages/components/stories/Dialog.css]: CSS in JSX looks really ugly :thinking:

Comment (frassinier) [reply packages/components/stories/Dialog.css]: Except if @glorieux does not like this.

Comment (frassinier) [reply packages/components/stories/Dialog.css]: BTW, @rmannibucau, my change request concerns the CSS and not where we have to put it, I can't click on modal buttons anymore with yours!

Comment (rmannibucau) [reply packages/components/stories/Dialog.css]: done

**Pull request merged**